### PR TITLE
New plugins Arrow Flight and Dremio 

### PR DESCRIPTION
**Pull request by iNikitaGricenko**
### What changes are being made and why?

Added Dremio jdbc plugin and Apache Arrow Flight. Splited both plugins as separated plugins
---

### How the changes have been QAed?
Trough Dremio test drive and Dremio account

```yaml
id : dremio-listen
namespace: dev

triggers:
  - id: watch
    type: io.kestra.plugin.jdbc.dremio.Trigger
    url: "jdbc:arrow-flight-sql://localhost:31010/?useEncryption=false"
    username: "dremio"
    password: "dremio123"
    sql: "SELECT * from \"dremio-test\".\"postgres.public\".departments"
    fetch: true
    interval: PT10S

tasks:
  - id: query
    type: io.kestra.plugin.jdbc.dremio.Query
    url: "jdbc:arrow-flight-sql://localhost:31010/?useEncryption=false"
    username: "dremio"
    password: "dremio123"
    sql: "SELECT * from \"dremio-test\".\"postgres.public\".departments"
    fetchOne: true
```
---

### Setup Instructions

- [Dremio Arrow Flight Authentication (URL example)](https://docs.dremio.com/current/sonar/client-applications/drivers/arrow-flight-sql-jdbc-driver/)

### CODE REVIEW

File: plugin-jdbc-arrow-flight/build.gradle [hunk 1]
```diff
@@ -0,0 +1,20 @@
+project.description = 'Query Dremio databases using the Kestra Apache Arrow Flight JDBC plugin.'
```
Comment (loicmathieu) [inline plugin-jdbc-arrow-flight/build.gradle:1]:
```suggestion
project.description = 'Query a compatible database using the Kestra Apache Arrow Flight SQL JDBC plugin.'
```

File: plugin-jdbc-arrow-flight/build.gradle [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+jar {
+    manifest {
+        attributes(
+                "X-Kestra-Name": project.name,
+                "X-Kestra-Title": "Arrow Flight",
```
Comment (loicmathieu) [inline plugin-jdbc-arrow-flight/build.gradle:7]:
```suggestion
                "X-Kestra-Title": "Arrow Flight SQL",
```

File: plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/Query.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+import java.sql.SQLException;
+import java.time.ZoneId;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Query a databases through Apache Arrow Flight SQL driver."
```
Comment (loicmathieu) [inline plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/Query.java:27]:
```suggestion
    title = "Query a database through Apache Arrow Flight SQL driver."
```

File: plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/Query.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Send a sql query to a Dremio direct database and fetch a row as outputs using Apache Arrow Flight SQL driver",
```
Comment (loicmathieu) [inline plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/Query.java:32]:
```suggestion
            title = "Send a SQL query to a database and fetch a row as outputs using Apache Arrow Flight SQL driver",
```

File: plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/Query.java [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+            code = {
+                "url: jdbc:arrow-flight-sql://localhost:31010/?useEncryption=false",
+                "username: dremio",
+                "password: dremio123",
+                "sql: select * FROM \"postgres.public\".departments",
```
Comment (loicmathieu) [inline plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/Query.java:37]:
```suggestion
                "sql: select * FROM departments",
```

File: plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/Trigger.java [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Wait for query on a Arrow Flight database."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Wait for a sql query to return results and iterate through rows",
```
Comment (loicmathieu) [inline plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/Trigger.java:30]:
```suggestion
            title = "Wait for a SQL query to return results and iterate through rows",
```

File: plugin-jdbc-dremio/build.gradle [hunk 7]
```diff
@@ -0,0 +1,20 @@
+project.description = 'Query Dremio databases using the Kestra JDBC plugin.'
```
Comment (loicmathieu) [inline plugin-jdbc-dremio/build.gradle:1]:
```suggestion
project.description = 'Query Dremio database using the Kestra JDBC plugin.'
```

File: plugin-jdbc-dremio/src/main/java/io/kestra/plugin/jdbc/dremio/Query.java [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Query a Dremio database."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Send a sql query to a Dremio database and fetch a row as outputs",
```
Comment (loicmathieu) [inline plugin-jdbc-dremio/src/main/java/io/kestra/plugin/jdbc/dremio/Query.java:31]:
```suggestion
            title = "Send a SQL query to a Dremio database and fetch a row as outputs",
```

File: plugin-jdbc-dremio/src/main/java/io/kestra/plugin/jdbc/dremio/Trigger.java [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Wait for query on a Dremio database."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Wait for a sql query to return results and iterate through rows",
```
Comment (loicmathieu) [inline plugin-jdbc-dremio/src/main/java/io/kestra/plugin/jdbc/dremio/Trigger.java:29]:
```suggestion
            title = "Wait for a SQL query to return results and iterate through rows",
```

File: plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/package-info.java [hunk 10]
```diff
@@ -0,0 +1,7 @@
+@PluginSubGroup(
+    description = "This sub-group of plugins contains tasks for accessing the databases through Apache Arrow Flight.",
```
Comment (loicmathieu) [inline plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/package-info.java:2]:
```suggestion
    description = "This sub-group of plugins contains tasks for accessing a databases through Apache Arrow Flight SQL.",
```

**Review by loicmathieu — CHANGES_REQUESTED** [review]
It is missing tests for Query and Trigger tasks.
In your test, please create a table with multiple row types (as done in other JDBC plugins) to check that the CellConverter didn't need to handle type conversions.

**Comment by anna-geller** [comment]
@iNikitaGricenko using this branch, I'm getting a bunch of errors on kestra server startup. see below. could you cross-check?

```yaml
2023-10-25 22:28:14,507 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.postgresql.Batch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-postgres-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,509 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-postgres-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,509 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.postgresql.Trigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-postgres-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,509 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-postgres-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,509 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.postgresql.Query' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-postgres-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,509 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.postgresql.CopyOut' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-postgres-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,509 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.postgresql.CopyIn' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-postgres-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,510 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.postgresql.AbstractCopy' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-postgres-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,510 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-postgres-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,514 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.vertica.Trigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-vertica-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,515 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.vertica.Batch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-vertica-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,515 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-vertica-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,515 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-vertica-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,515 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-vertica-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,515 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.vertica.Query' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-vertica-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,622 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-clickhouse-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,622 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.clickhouse.Trigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-clickhouse-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,622 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-clickhouse-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,623 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.clickhouse.Query' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-clickhouse-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,623 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-clickhouse-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,625 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,625 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,625 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,647 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-arrow-flight-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,648 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-arrow-flight-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,648 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.arrowflight.Trigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-arrow-flight-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,648 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.arrowflight.Query' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-arrow-flight-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,648 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-arrow-flight-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,677 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.sqlserver.Trigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-sqlserver-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,677 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-sqlserver-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,677 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.sqlserver.Query' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-sqlserver-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,677 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-sqlserver-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,677 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-sqlserver-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,677 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.sqlserver.Batch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-sqlserver-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,800 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-pinot-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,801 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-pinot-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,801 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-pinot-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,801 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.pinot.Trigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-pinot-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,801 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.pinot.Query' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-pinot-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,851 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-duckdb-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,852 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.duckdb.Trigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-duckdb-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,852 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-duckdb-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,852 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.duckdb.Query' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-duckdb-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,852 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-duckdb-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,883 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-trino-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,884 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.trino.Query' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-trino-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,884 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-trino-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,884 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.trino.Trigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-trino-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,884 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-trino-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,920 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-oracle-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,921 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.oracle.Query' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-oracle-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,921 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-oracle-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,921 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-oracle-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,921 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.oracle.Batch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-oracle-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,921 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.oracle.Trigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-oracle-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,954 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.vectorwise.Query' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-vectorwise-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,954 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-vectorwise-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,954 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-vectorwise-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,954 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.vectorwise.Batch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-vectorwise-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,955 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-vectorwise-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,955 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.vectorwise.Trigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-vectorwise-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,992 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-mysql-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,992 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-mysql-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,993 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.mysql.Query' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-mysql-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,993 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-mysql-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:14,993 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.mysql.Trigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-mysql-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,121 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-snowflake-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,122 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.snowflake.AbstractSnowflakeConnection' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-snowflake-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,122 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-snowflake-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,122 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-snowflake-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,122 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.snowflake.Download' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-snowflake-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,122 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.snowflake.Query' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-snowflake-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,122 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.snowflake.Upload' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-snowflake-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,123 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.snowflake.Trigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-snowflake-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,177 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.dremio.Trigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-dremio-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,177 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-dremio-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,177 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-dremio-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,178 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-dremio-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,178 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.dremio.Query' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-dremio-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,224 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-redshift-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,224 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.redshift.Trigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-redshift-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,224 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-redshift-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,224 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-redshift-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,225 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.redshift.Query' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-redshift-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,229 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcQuery' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-rockset-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,230 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcBatch' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-rockset-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,230 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.rockset.Query' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-rockset-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,230 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.AbstractJdbcTrigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-rockset-0.13.0-SNAPSHOT.jar'
2023-10-25 22:28:15,230 WARN  main         io.kestra.core.plugins.PluginScanner Unable to load class 'io.kestra.plugin.jdbc.rockset.Trigger' on plugin 'file:/Users/anna/dev/plugins/plugin-jdbc-rockset-0.13.0-SNAPSHOT.jar'
```

also, did you QA the Arrow Flight implementation with Dremio? if so, could you update the PR body with an Arrow example that worked for you with a Dremio Test Drive account?

**Comment by iNikitaGricenko** [comment]
@anna-geller, databases that are given into Dremio Test Drive does not support apache arrow flight

**Comment by anna-geller** [comment]
I'm sorry, I didn't know that.

**Pull request merged**